.github/scripts/setup_xspec.sh

```diff
@@ -20,6 +20,9 @@ xspec_library_path=${xspec_root}/lib/
 xspec_include_path=${xspec_root}/include/
 
 case "${XSPECVER}" in
+  12.13.1*)
+      xspec_version_string="12.13.1"
+      ;;
   12.13.0*)
```

Comment:
Can we drop either the 12.12.0 or 12.12.1 rules (just below) now?

Reply:
I don't have any preference, so I'm fine with dropping either/both from here and the test matrix if we don't need it

Reply:
If we don't use them in the test matrix then I suggest dropping them (we don't want to be guaranteeing to users that we'll keep the old versions around!).

Reply:
If that's the case, are you okay with this going in for now and a separate PR being made for dropping them from the test matrix, this file, and any other doc/source updates?

Reply:
Yeah - let's get this done now so our CI runs get to exercise this.